### PR TITLE
Add feature to copy required driver files to specified locations in the api container

### PR DESCRIPTION
**Pull request by jhazentia**

### CODE REVIEW

File: drivers/README.md [hunk 1]
```diff
@@ -1,8 +1,34 @@
...
-within the SCOS sensor Docker container.
+Some signal analyzers require drivers which cannot be packaged with their respective
+SCOS plugins. Those drivers can be placed here manually, allowing them to be referenced
+from within the scos-sensor Docker container.
+
+A json file can be used to copy files in this directory to a required destination in
+the Docker container.
+
+Below is a sample json file. The `"source_path"` must be relative to the `drivers`
+directory. The `"dest_path"` can be anywhere in the Docker container.
```
Comment (dboulware) [inline drivers/README.md:14]: I assume dest_path has to be a full path?

File: src/sensor/utils.py [hunk 2]
```diff
@@ -61,4 +66,35 @@ def get_timestamp_from_datetime(dt):
...
+                            settings.DRIVERS_DIR, scos_file["source_path"]
+                        )
+                        if not os.path.isfile(source_path):
+                            logger.error(f"Unable to find file at {source_path}")
+                            continue
+                        dest_path = scos_file["dest_path"]
+                        dest_dir = os.path.dirname(dest_path)
+                        try:
+                            if not os.path.isdir(dest_dir):
+                                os.makedirs(dest_dir)
```
Comment (dboulware) [inline src/sensor/utils.py:95]: Will this fail for nested directories?

File: drivers/README.md (hunk 1 above)
Comment (dboulware) [inline drivers/README.md:14]: I would add a note that the directory will be created if it does not exist and indicate whether or not the parent directory must already exist or not. See question in utils.py for context.

**Review by dboulware — CHANGES_REQUESTED** [review]
See Readme comment and utils.py question.

File: src/sensor/utils.py (hunk 2 above)
Comment (jhazentia) [reply src/sensor/utils.py:95]: `makedirs` will make parent directories: https://docs.python.org/3/library/os.html#os.makedirs

**Review by dboulware — APPROVED** [review]

**Pull request merged**